docker/nginx/nginx.conf

```diff
@@ -0,0 +1,31 @@
+user  www-data;
```

Comment:
Do you think we can merge both configurations?

https://github.com/Sylius/Sylius-Standard/blob/1.12/docker/nginx/conf.d/default.conf

if they can be separate, then I would rather have them moved to conf.d directory and let nginx handle loading it

Reply:
The files from `conf.d` are included in the `http` section, they can't be merged and this file can't go to `conf.d`.

Reply:
It's how docs recommend to customize it: https://github.com/docker-library/docs/tree/master/nginx#complex-configuration